bears/rest/RSTcheckBear.py

```diff
@@ -0,0 +1,58 @@
+import re
+
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class RSTcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck', '2.*')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/8ntlaqubk2qkrn9mm0dh07rlk?speed=2'
+
+    def run(self, filename, file,
+            code_block_language_ignore: list = ()):
+        """
+        A Bear for rstcheck. Check
+        <https://pypi.python.org/pypi/rstcheck> for more information on
+        rstcheck.
+        Rstcheck checks syntax of reStructuredText and code blocks
+        nested within it.
+        Supports code blocks of ``bash``, ``c``, ``cpp``, ``json``,
+        ``python`` and ``rst``.
+
+        :param code_block_language_ignore:
+            Comma seperated list for languages of which code blocks
+            should be ignored. Code block of following languages
+            can be detected: ``bash``, ``c``, ``cpp``, ``json``,
+            ``python``, ``rst``.
+        """
+        for error in check(''.join(file), filename=filename,
+                           ignore=code_block_language_ignore):
+
+            try:
+                report_level = int(
+                    re.search(r'\(\w+\/([0-9])\)', error[1]).group(1))
```

Comment:
If I understand correctly we are matching `(INFO/1)` or `(ERROR/3)`, in which case this regex is not very precise, and could be more 'literate'  - it should match the `INFO` in `(INFO/..)`, rather than matching the number.  Also the number range is not `[0-9]` - that will lead to errors - see below.